GitUI/UserControls/RevisionGrid/FormRevisionFilter.cs

```diff
@@ -149,6 +149,12 @@ public void SetBranchFilter(string filter)
             BranchFilter.Text = filter?.Trim();
         }
 
+        public void SetPathFilter(string filter)
+        {
+            FileFilterCheck.Checked = !string.IsNullOrWhiteSpace(filter);
+            FileFilter.Text = filter?.Trim();
```

Comment:
Have you considered filenames with spaces at start/end?

Reply:
GE added filenames (for FileHistory and a later PR for a context menu for RevDiff etc) are quouted when adding.
User added paths in Advanced filter must be correctly quoted by the user.
(There is a helper for quoting paths without spaces and quotes, likely the most common case).

> A separate commit with the renaming would ease the review a little.

This was separated in several commits until I squashed it preparing for merge.
But commits are still separate in branch gerhardol/feature/i7598-replace-formfilehistory